silverback/_cli.py

```diff
@@ -14,7 +14,7 @@ def cli():
 
 
 @cli.command(cls=NetworkBoundCommand)
-@ape_cli_context()
+@verbosity_option()
```

Comment:
one note, if silverback ever gets its own logger, you can put that here to use instead of ape's